pylint/test/unittest_reporters_json.py

```diff
@@ -34,6 +34,7 @@ def test_simple_json_output():
         ("column", 0),
         ("line", 1),
         ("message", "Line too long (1/2)"),
+        ("message-id", "C0301"),
```

Comment:
IIRC issue mentioned reporting _symbolic message_ - so in this case it would be `line-too-long`.